app/src/main/java/org/catroid/catrobat/newui/ui/fragment/LookListFragment.java

```diff
@@ -103,4 +110,20 @@ private FileInfo createImage() {
 
         return new FileInfo(Utils.getImageDirectory(), file.getName());
     }
+    @Override
+    public boolean isNameValid(String itemName){
+        if(itemName.equals(""))
+            return false;
+        return Utils.isItemNameUnique(itemName, mRecyclerViewAdapter.getItems());
+
+    }
+
+    @Override
+    public void renameItem(String itemName) {
+        List<LookInfo> item = mRecyclerViewAdapter.getSelectedItems();
+        if(item.size() == 1) {
```

Comment:
this check is unnecessary, as there can only be one item selected for renaming. if for any reason there happens to be more than one item this should be handled in a way that the user is informed about the error (doesn't have to be handled now though).